### PR TITLE
fix(geo): solve absence of legend when printing an image

### PR DESCRIPTION
**Pull request by aziz-access**
**What is the current behavior?** (You can also link to an open issue here)

this update relate to the issue [678](https://github.com/infra-geo-ouverte/igo2/issues/678)  of igo2

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [ ] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:

### CODE REVIEW

File: packages/geo/src/lib/print/print/print.component.ts [hunk 1]
```diff
@@ -125,7 +126,7 @@ export class PrintComponent {
           data.title,
           data.subtitle,
           data.comment,
-          data.doZipFile
+          data.doZipFile,
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/print/print/print.component.ts:129]: remove ,

File: packages/geo/src/lib/print/print-form/print-form.component.ts [hunk 2]
```diff
@@ -224,6 +224,9 @@ export class PrintFormComponent implements OnInit {
 
   handleFormSubmit(data: PrintOptions, isValid: boolean) {
     data.isPrintService = this.isPrintService;
+    if(data.legendPosition === 'newpage') {
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/print/print-form/print-form.component.ts:227]: Add for every legend position

**Review by PhilippeLafreniere18 — APPROVED** [review]

File: packages/geo/src/lib/print/print-form/print-form.component.ts (hunk 2 above)
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/print/print-form/print-form.component.ts:227]: Add for every legend position

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]
There is two problems i've seen.

1- When the legend is added on a new page, It is only the legend that is printed with the image (all formats does that).

2- Not directly link to this specific issue, but title, subtitle, comments, projection and scale aren't printed with all image formats except for JPEG who works fine.

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]
Change translation when adding the legend on a new image (instead of page)

**Review by PhilippeLafreniere18 — APPROVED** [review]

**Comment by pelord** [comment]
https://github.com/infra-geo-ouverte/igo2/issues/913 into the next version.

**Pull request merged**